server/index.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const morgan = require('morgan');
+const db = require('../database/index');
+
+
+const app = express();
+app.use(bodyParser.json());
+app.use(morgan('dev'));
+
+let staticPath = __dirname + '/../public';
+console.log(staticPath);
+app.use('/books/:id', express.static(staticPath));
```

Comment:
I believe it will be more appropriate to use the index route "/" to serve the static files. Because the index will serve as the entry point to the application and new users may not be able to know any other route. I will fix mine too.

Reply:
I think the "/" route should be theoretically the main goodreads page. The static file we serve should be an HTML page showing ONE book item with a specific id. So the static route should be "/book/:id"